Core/SubscriptionManager/src/main/resources/application.yml

```diff
@@ -22,7 +22,10 @@ csource:
     topic: CSOURCE_REGISTRATION
   notification:
     topic: CONTEXT_SOURCE_NOTIFICATION
-
+query:
```

Comment:
[Minor]indentation is not proper. it should be same as notification(above one) level.